File: src/main/java/eu/mjindra/utils/dices/D10.java
package eu.mjindra.utils.dices;

/**
 * A D10 die.
 *
 * @author Martin Jindra
 * @version 31.05.2023
 */
public class D10 extends Dice {

    public D10(byte amount) {
        this(amount, (byte) 10);
    }

    public D10(byte amount, byte modifier) {
        super(amount, (byte) 10, modifier);
    }

}

File: lib/dnd/dices/d10.dart
import 'package:helios/dnd/dices/dice.dart';
import 'package:helios/dnd/dices/die.dart' show Die;

/// A D10 die.
class D10 extends Die {
  D10() : super(DiceSides.d10);

  D10.mod(int modifier) : super.mod(DiceSides.d10, modifier);
}
